test/test_torch.py

```diff
@@ -14226,6 +14226,23 @@ def test_min_max_binary_op_nan(self, device, dtype):
             self.assertFalse(torch.isnan(ma[i]), "max(a, b): {}, a: {}, b: {}".format(ma[i], a[i], b[i]))
             self.assertFalse(torch.isnan(mi[i]), "min(a, b): {}, a: {}, b: {}".format(mi[i], a[i], b[i]))
 
+    @dtypesIfCUDA(torch.half, torch.float, torch.double)
+    @dtypes(torch.float, torch.double)(
+    def test_min_max_empty_element(self, device, dtype):
+        a = torch.rand(0, dtype=dtype, device=device)
+        b = torch.rand(0, 5, dtype=dtype, device=device)
+        c = torch.rand(0, 5, 10, dtype=dtype, device=device)
+        d = torch.rand(0, 5, 10, 15, dtype=dtype, device=device)
+        e = torch.rand(0, 5, 10, 15, 20, dtype=dtype, device=device)
+
+        ident_err = 'operation does not have an identity'
+        self.assertRaisesRegex(RuntimeError, ident_err, a.max())
```

Comment:
There are other cases where you should get a RunTimeError that you need to check. The following NumPy snippet may suggest some more test cases:

```
a = np.random.rand(1, 0) 
a.max(0)
a.max(1)
```